supercommand.go

```diff
@@ -101,7 +101,8 @@ type SuperCommandParams struct {
 	// will use that name when referring to an individual items/flags in this command.
 	// For example, if this value is 'option', the default message 'value for flag'
 	// will become 'value for option'.
-	FlagKnownAs string
+	FlagKnownAs             string
+	FlagDocumentationHidden bool
```

Comment:
I don't think we should call this `FlagX`, as it could be confused with "flags", as in the `--options` passed to a command. Why not just `ExportDocs bool` ?

P.S. let's add a doc comment here explaining what this value does - including the fact that it's only used in the `juju documentation` command.

Reply:
It's only hiding the super command docs. So maybe `IgnoreCommandDoc` or something similar?